packages/did/src/utils/parser.ts

```diff
@@ -0,0 +1,34 @@
+export const didPattern = `^did:[a-z0-9]+?:?[a-z0-9]+?:(0x[A-Fa-f0-9]{40})$`;
+
+/**
+* Checks if did is valid and returns the did method
+*
+* @param did
+* @private
+*/
+export function getDIDMethod(did : string): string {
```

Comment:
Might as well return one of the `Methods` from `../models`, I think

Reply:
Yeah, could be done. But then we will have to match all the methods available(we might have plenty of methods in future for our chains). I guess it's better to return just the `method` and the calling function could validate it at it's end.

Reply:
Ok, fair enough 👍